[datadog_app_builder_app] Add App Builder Terraform Support (#2723)
DataDog/terraform-provider-datadog: docs/resources/app.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "datadog_app Resource - terraform-provider-datadog"
subcategory: ""
description: |-

>   Provides a Datadog App resource. This can be used to create and manage a Datadog App from the App Builder product.

Reviewer: ```suggestion
  Provides a Datadog App resource for creating and managing Datadog Apps from App Builder.
```

---

# datadog_app (Resource)

> Provides a Datadog App resource. This can be used to create and manage a Datadog App from the App Builder product.

Reviewer: ```suggestion
Provides a Datadog App resource for creating and managing Datadog Apps from App Builder.
```

## Example Usage

```terraform

> # Example app json - loaded from a file (necessary if includes ${} syntax)

Reviewer: ```suggestion
# Example app JSON - loaded from a file (required for ${} syntax)
```
